bin/build.sh

```diff
@@ -221,21 +226,38 @@ then
     sudo zypper --non-interactive install ${DEPENDENCIES}
     ;;
 
+  linux-neon-1[6]*)
```

Comment:
Could this case possibly be merged with the following `linux-ubuntu` case? The only differences seem to be that `xential` replaces `trusty` in the LLVM apt repo URL. Merging might be achieved by adding a release-name table mapping `neon` to `xenial` and `ubuntu` to `trusty`.